recipe/data.go

```diff
@@ -1753,19 +1753,19 @@ var AgeingSingleHouseholds = Response{
 	},
 }
 
-// Local authority ageing statistics - sex ratios for people over 65 and over 85
+// Local authority ageing statistics, sex ratios for people aged 65 and over and 85 and over
 var AgeingSexRatios = Response{
 	ID:     "df8c8df5-484a-4d29-b246-a7720caf2474",
-	Alias:  "Local authority ageing statistics - sex ratios for people over 65 and over 85",
+	Alias:  "Local authority ageing statistics, sex ratios for people aged 65 and over and 85 and over",
 	Format: "v4",
 	InputFiles: []file{
 		{"AgeingSexRatios"},
 	},
 	OutputInstances: []instance{
 		{
-			DatasetID: "ageing-sex-ratios",
+			DatasetID: "older-people-sex-ratios",
 			Editions:  []string{"time-series"},
-			Title:     "Local authority ageing statistics, sex ratios for people over 65 and over 85",
+			Title:     "Local authority ageing statistics, sex ratios for people aged 65 and over and 85 and over",
```

Comment:
Should there be a comma here:
```suggestion
			Title:     "Local authority ageing statistics, sex ratios for people aged 65 and over, and 85 and over",
```

Reply:
same as previous